mlproject/unicore-config-venv.yaml

```diff
@@ -0,0 +1,22 @@
+UnicoreApiUrl: https://zam2125.zam.kfa-juelich.de:9112/JUWELS/rest/core
+Environment:
+  Python: /p/project/deepacf/maelstrom/grau1/ap3_env
+  Modules:
+    - Stages/2022
+    - GCCcore/.11.2.0
+    - GCC/11.2.0
+    - cuDNN/8.3.1.22-CUDA-11.5
+    - Python/3.9.6
+  Variables:
+    GIT_PYTHON_REFRESH: quiet
+Resources:
+  Queue: develbooster
+  Nodes: 1
+  Project: deepacf
```

Comment:
This is actually not read and passed, is it? Afaik, this is read from `MANTIK_COMPUTE_BUDGET_ACCOUNT` when using the CLI.